### ISSUE TITLE
Provide accessible error messages for AT users

### ISSUE THREAD

**Issue by dmstern** [rjsf-team/react-jsonschema-form#959]
### Prerequisites

- [x] I have read the [documentation](https://github.com/mozilla-services/react-jsonschema-form/blob/master/README.md#readme);
- [x] In the case of a bug report, I understand that providing a [SSCCE](http://sscce.org/) example is tremendously useful to the maintainers.

### Description

In order to providy more accessibility to our screen reaser users, we would like to add an `aria-describedby`-attribute to input fields that reference the inline error message (e.g. "is a required property") with an id of this error meesage when validating the form. This would cause the AT to read out the error message under the input field when the user tabs through the form elements.

Is there a way to configure `aria-describedby`-attributes and to enable ids to error messages?

Any other ideas to solve this problem are appreciated. :)

For example, instead of this markup from the [live playground](https://mozilla-services.github.io/react-jsonschema-form/)

```html
<div class="form-group field field-string field-error has-error has-danger">
  <label class="control-label" for="root_lastName">Last name*</label>
  <div></div>
  <input type="text" 
     class="form-control" 
     value="" 
     id="root_lastName" 
     label="Last name" 
     required="" 
     placeholder="">
  <div>
    <p></p>
    <ul class="error-detail bs-callout bs-callout-info">
      <li class="text-danger">is a required property</li>
    </ul>
  </div>
  <div></div>
</div>
```

it should look like this:
```html
<div class="form-group field field-string field-error has-error has-danger">
  <label class="control-label" for="root_lastName">Last name*</label>
  <div></div>
  <input type="text" 
     class="form-control" 
     value="" 
     id="root_lastName" 
     label="Last name" 
     required="" 
     placeholder="" 
<!-- CHANGE -->
     aria-describedby="error-message-123456789" 
<!-- /CHANGE -->
     >
  <div>
    <p></p>
    <ul class="error-detail bs-callout bs-callout-info">
      <li class="text-danger" 
<!-- CHANGE -->
          id="error-message-123456789"
<!-- /CHANGE -->
          >is a required property</li>
    </ul>
  </div>
  <div></div>
</div>
```

### Steps to Reproduce

1. Install a screen reader like [NVDA](https://www.nvaccess.org/download/)
2. Launch the screen reader
3. Open the [live playground](https://www.nvaccess.org/download/) in your browser (The first input field ('First Name') should be focused)
4. Use the <kbd>TAB</kbd> key to navigate to the required 'Last Name'-input-field.
5. Delete the value --> The field gets red and a message appears below the input field ('is a required property')
6. Listen to the screen reader.

#### Expected behavior

The screen reader should read out also the message that is configured here ("is a required property").

#### Actual behavior

The screen reader tells you that this field is required and should be filled, since it recognizes the HTML5 validation functionality of the browser. But the message is not being read out.

**Comment by epicfaace** [comment]
This is important, we should do this
